celerybeatmongo/schedulers.py

```diff
@@ -56,6 +56,9 @@ def next(self):
     def is_due(self):
         if not self._task.enabled:
             return False, 5.0   # 5 second delay for re-enable.
+        if hasattr(self._task, 'max_run_count') and self._task.max_run_count:
```

Comment:
Now that `max_run_count` is defined on the model, is there any reason that it would not be defined?